src/main/java/eu/enexa/docker/ContainerManagerImpl.java

```diff
@@ -56,6 +56,13 @@ public ContainerManagerImpl(){
         LOGGER.info("docker client is pinged");
     }
 
+    public ContainerManagerImpl(String sharedDirectory, String experimentWriteablePathDirectory, String modulePathDirectory){
```

Comment:
A short Javadoc comment at least for the public methods that do not override a (hopefully already documented) method of a super class or interface would be nice :wink: